benchmarks/gpu_pyfixest_errors.ipynb

```diff
@@ -0,0 +1,1478 @@
+{
```

Comment:
<p>demeaning times ara nan?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Only when the demeaning is explicitly skipped.